src/js/actions/deploymentActions.js

```diff
@@ -30,7 +30,10 @@ export const getDeployments = (page = default_page, per_page = default_per_page)
         accu[item.status].push(item);
         return accu;
       },
-      { finished: [], inprogress: [], pending: [] }
+      Object.keys(getState().deployments.byStatus).reduce((accu, item) => {
+        accu[item] = [];
+        return accu;
+      }, {})
```

Comment:
real nitpicking, this is used twice, could be a separate util, `emptyStatusHash`

Reply:
argh, yep - that's annoying... and sounds like I'll have 2 small PRs to follow up with 😄!

Reply:
https://github.com/mendersoftware/gui/pull/690

Reply:
thanks. approved.